tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -995,3 +999,94 @@ def create_and_validate_custom_host(node_roles, random_cluster_name=False):
         i += 1
     cluster = validate_cluster(client, cluster, k8s_version=K8S_VERSION)
     return cluster, aws_nodes
+
+
+def test_kdm_changes():
+    k8s_version_url = CATTLE_API_URL + K8S_VERSION_URL
+    head = {'Authorization': 'Bearer ' + USER_TOKEN}
+    response = requests.get(k8s_version_url, verify=False, headers=head)
+    data = response.json()
+    k8s_v = data["default"]
+    default_k8s_versions = k8s_v.split(",")
+    plugin = ["calico", "canal", "flannel"]
+    list_process = []
+    cluster_list = []
+    for k8s_version in default_k8s_versions:
+        rke_config_new = deepcopy(rke_config)
+        rke_config_new["kubernetesVersion"] = k8s_version
+        node_count = NODE_COUNT_KDM_CLUSTER * len(plugin)
+        aws_nodes = \
+            AmazonWebServices().create_multiple_nodes(
+                node_count, random_test_name(HOST_NAME))
+        i = 0
+        for plug in plugin:
+            if plug == "calico" or plug == "canal":
+                rke_config_new["network"]["options"] = \
+                    {"flannel_backend_type": "vxlan"}
+            rke_config_new["network"] = {"type": "networkConfig",
+                                         "plugin": plug}
+            print(rke_config_new["network"]["plugin"])
+            print(rke_config_new["kubernetesVersion"])
+            client = get_user_client()
+            cluster_name = random_name()
+            cluster = client.create_cluster(name=cluster_name,
+                                            driver="rancherKubernetesEngine",
+                                            rancherKubernetesEngineConfig=
+                                            rke_config_new)
+            p1 = multiprocessing.Process(target=validate_custom_cluster_kdm,
+                                         args=(cluster,
+                                               rke_config_new,
+                                               aws_nodes
+                                               [i:i+NODE_COUNT_KDM_CLUSTER]))
+            cluster_list.append(cluster)
+            list_process.append(p1)
+            p1.start()
+            i = i + NODE_COUNT_KDM_CLUSTER
+    for process in list_process:
+        process.join()
+    '''
+    setting cluster count to an environment 
+    variable to be read by a jenkins job
+    '''
+    cluster_count = len(cluster_list)
+    i = 0
+    # writing details into a file
+    env_details = "env.CLUSTER_COUNT_KDM_VALIDATION='" + cluster_count + "'\n"
+    env_details = "env.CATTLE_TEST_URL='" + CATTLE_TEST_URL + "'\n"
```

Comment:
should it be `+=` instead of `=`? It overwrites the value from the previous line.